rasa_core/policies/fallback.py

```diff
@@ -47,6 +47,7 @@ def __init__(self,
         self.nlu_threshold = nlu_threshold
         self.core_threshold = core_threshold
         self.fallback_action_name = fallback_action_name
+        self.core_threshold_set = False
```

Comment:
Do we need this in the initialization here if it's only used in the `predict_action_probabilities`?

Reply:
As we are accessing the variable outside the `FallbackPolicy` class in the ensemble, we need the initialization.